pages.zh_TW/common/code.md

```diff
@@ -0,0 +1,28 @@
+# code
+
+> Visual Studio Code.
```

Comment:
Can his not be translated

Reply:
Does the Chinese version use a word instead of Visual Studio Code

Reply:
I've hardly ever seen the words "visual studio code" written in Chinese. I can add the Chinese translation, but we usually mention it in English words.